boards/arm/cc3220sf_launchxl/cc3220sf_launchxl_connector.dtsi

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022 Dhruva Gole
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	launchxl_connector: connector {
+		compatible = "launchxl-connector";
```

Comment:
When introducing a new binding, you need to add the matching yaml binding

Reply:
I'm sorry, I do not quite follow? Can you show me an example?
I tried taking a look at the ``arduino_nano_33_ble.yaml`` and couldn't find any "new binding" pertaining to the connector.dtsi.

Reply:
see dts/bindings/gpio/arduino-header-r3.yaml

Reply:
Edit: When introducing a new ~~binding~~compatible, you need to add the matching yaml binding

Reply:
okay @erwango thanks for pointing out, converting to draft PR till that yaml is done.

Reply:
Also to be 100% sure I'll try to test if every gpio-map I have done is working alright or not.
Is this followed across zephyr or is there some other way to validate more smart-ly?